modules/client/src/interfaces.ts

```diff
@@ -49,7 +51,7 @@ export interface IClientEncoding extends IClientCore {
   ) => Promise<DaoAction>;
   updateMetadataAction: (
     daoAddressOrEns: string,
-    params: IMetadata,
+    params: string,
```

Comment:
Does the string contain the new URI? If so, we should name it accordingly.

Do we have the equivalent on DAO creation?